packages/firebase_crashlytics/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: firebase_crashlytics
 description: Flutter plugin for Firebase Crashlytics. It reports uncaught errors to the
   Firebase console.
-version: 0.0.4+12
+version: 0.0.4+13
```

Comment:
I guess the version has to be bumped to `0.5` to signal the breaking change?

Reply:
It should be 0.0.5

Reply:
Done